src/components/FarmCard/FarmCardDetails.tsx

```diff
@@ -278,125 +292,206 @@ const FarmCardDetails: React.FC<{
+                  mt={2}
+                  p={2}
+                  onClick={async () => {
+                    if (
+                      !approving &&
+                      !attemptStaking &&
+                      Number(stakeAmount) > 0 &&
+                      Number(stakeAmount) <=
+                        getExactTokenAmount(userLiquidityUnstaked)
+                    ) {
+                      if (approval === ApprovalState.APPROVED) {
+                        onStake();
+                      } else {
+                        onAttemptToApprove();
+                      }
+                    }
+                  }}
+                >
+                  <Typography variant='body1'>
+                    {attemptStaking
+                      ? 'Staking LP Tokens...'
+                      : approval === ApprovalState.APPROVED
+                      ? 'Stake LP Tokens'
+                      : approving
+                      ? 'Approving...'
+                      : 'Approve'}
+                  </Typography>
                 </Box>
               </Box>
-              <Box className={classes.inputVal} mb={2} mt={2} p={2}>
+            )}
+            <Box className={classes.buttonWrapper} mx={isMobile ? 0 : 2} my={2}>
+              <Box display='flex' justifyContent='space-between'>
+                <Typography variant='body2'>My deposits:</Typography>
+                <Typography variant='body2'>
+                  {formatTokenAmount(stakingInfo.stakedAmount)} LP{' '}
+                  <span>({getUSDString(stakedAmounts?.myStakedUSD)})</span>
+                </Typography>
+              </Box>
+              <Box className={classes.inputVal} mb={2} mt={4.5} p={2}>
                 <NumericalInput
                   placeholder='0.00'
-                  value={stakeAmount}
+                  value={unstakeAmount}
                   fontSize={16}
                   onUserInput={(value) => {
-                    setStakeAmount(value);
+                    setUnStakeAmount(value);
                   }}
                 />
                 <Typography
                   variant='body2'
                   style={{
                     color:
-                      userLiquidityUnstaked &&
-                      userLiquidityUnstaked.greaterThan('0')
+                      stakingInfo.stakedAmount &&
+                      stakingInfo.stakedAmount.greaterThan('0')
                         ? palette.primary.main
                         : palette.text.hint,
                   }}
                   onClick={() => {
                     if (
-                      userLiquidityUnstaked &&
-                      userLiquidityUnstaked.greaterThan('0')
+                      stakingInfo.stakedAmount &&
+                      stakingInfo.stakedAmount.greaterThan('0')
                     ) {
-                      setStakeAmount(userLiquidityUnstaked.toExact());
+                      setUnStakeAmount(stakingInfo.stakedAmount.toExact());
```

Comment:
Same comment as above for the code in this area just want to make sure it was intentional

Reply:
I did not update this part. I think it looks like it is updated because of previous updates. You can check this screenshot.
<img width="488" alt="screenshot" src="https://user-images.githubusercontent.com/47202087/161663627-c56f7a63-f81f-43e2-9865-dda4cded38b1.png">